tests/docstrings/T14.env

```
@@ -0,0 +1 @@
+CRYPTOL_SAW=sh ./mocksaw.sh
```

I think this needs to be quoted:

```suggestion
CRYPTOL_SAW="sh ./mocksaw.sh"
```